inference-engine/src/transformations/src/transformations/convert_quantize_dequantize.cpp

```diff
@@ -0,0 +1,139 @@
+//                  +----------+    +------------+
+//                  |zero point|--->|  Subtract  |
+//                  +----------+    +-----+------+
+//                                        |
+//                                        v
+//                   +---------+    +------------+
+//                   |  scale  |--->|  Multiply  |
+//                   +---------+    +-----+------+
+//                                        |
+//                                        v
+//
+
+
+ngraph::pass::ConvertQuantizeDequantize::ConvertQuantizeDequantize() {
+    auto data_pattern = ngraph::pattern::any_input();
+    auto input_low_pattern = ngraph::pattern::any_input();
+    auto input_high_pattern = ngraph::pattern::any_input();
+    auto output_low_pattern = ngraph::pattern::wrap_type<opset1::Constant>();
+    auto output_high_pattern = ngraph::pattern::wrap_type<opset1::Constant>();
+    auto fq_pattern = ngraph::pattern::wrap_type<opset1::FakeQuantize>({data_pattern, input_low_pattern,
+                                                                       input_high_pattern, output_low_pattern,
+                                                                       output_high_pattern});
+    auto convert1_pattern = ngraph::pattern::wrap_type<opset1::Convert>({fq_pattern}, pattern::type_matches_any({element::i8, element::u8}));
+    auto convert2_pattern = ngraph::pattern::wrap_type<opset1::Convert>({convert1_pattern}, pattern::type_matches(element::f32));
+    auto zero_point_pattern = ngraph::pattern::any_input();
+    auto sub_pattern = ngraph::pattern::wrap_type<opset1::Subtract>({convert2_pattern, zero_point_pattern}, pattern::consumers_count(1));
+    auto scale_pattern = ngraph::pattern::any_input();
+    auto mul_pattern = ngraph::pattern::wrap_type<opset1::Multiply>({sub_pattern, scale_pattern});
+
+    ngraph::graph_rewrite_callback callback = [=](pattern::Matcher& m) {
+        auto pattern_map = m.get_pattern_map();
+        auto data = pattern_map[data_pattern];
+        auto input_low = pattern_map[input_low_pattern];
+        auto input_high = pattern_map[input_high_pattern];
+        auto output_low = std::dynamic_pointer_cast<opset1::Constant>(pattern_map[output_low_pattern]);
+        auto output_high = std::dynamic_pointer_cast<opset1::Constant>(pattern_map[output_high_pattern]);
+        auto fq = std::dynamic_pointer_cast<opset1::FakeQuantize>(pattern_map[fq_pattern]);
+        auto zero_point = pattern_map[zero_point_pattern];
+        auto scale = pattern_map[scale_pattern];
+        auto convert1 = pattern_map[convert1_pattern];
+        auto convert2 = pattern_map[convert2_pattern];
+        auto sub = pattern_map[sub_pattern];
+        auto mul = pattern_map[mul_pattern];
+
+        size_t levels = fq->get_levels();
+        if (levels != 256)
+            return false;
+
+        float out_low_val;
+        if (!op::util::get_single_value(output_low, out_low_val))
```

Comment:
Please add a comment for statements that leads to the completion of the transformation execution